MODEL_SETUP.jl

```diff
@@ -375,6 +377,12 @@ elseif planet=="Venus"
                                     :O2=>Dict("n"=>[3e-3*ntot_at_lowerbdy, NaN], "f"=>[NaN, 0.]),
                                     :N2=>Dict("n"=>[0.032*ntot_at_lowerbdy, NaN]),
 
+                                    #Krasnopolsky, 2010a: this was 400ppb at 74km in altitude, and the actual number is likely lower (is either 4.0E-7, or 4.8E-7 depending on the calculation); and according to Zhang 2012 it is 3.66e-7
+                                    :HCl=>Dict("n"=>[3.66e-7 * ntot_at_lowerbdy, NaN]),
```

Comment:
This looks good, it also agrees with https://www.sciencedirect.com/science/article/pii/S0032063314004061?via%3Dihub which is a more recent paper and probably the right one to use (see Figure 5, I'm looking in between the 0-30 and 30-70 lat lines)